README.md

```diff
@@ -780,6 +781,20 @@ const fn = (one: string, two: string) => `${one} ${two}`;
 export default moize.maxSize(5)(fn);
 ```
 
+## moize.profile
```

Comment:
It seems this was missing in the existing documentation, so I've added it for completeness.